smarts/sstudio/types.py

```diff
@@ -304,6 +304,17 @@ def __eq__(self, other):
         return self.__class__ == other.__class__ and hash(self) == hash(other)
 
 
+@dataclass
+class Via:
+    """A point on an edge that an actor must pass through"""
+
+    edge_id: str
+    lane_index: int
+    lane_offset: int
+    required_speed: float
+    hit_distance: float = -1
```

Comment:
Can we add a comment what this means. How about using `None` over -1, but what does no hit distance even mean?

Reply:
How about `hit_radius`?